matter_server/common/custom_clusters.py

```diff
@@ -243,6 +245,52 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
 
             value: float32 = 0
 
+        @dataclass
+        class Altitude(ClusterAttributeDescriptor, CustomClusterAttributeMixin):
+            """Altitude Attribute within the Eve Cluster."""
+
+            should_poll = False
+
+            @ChipUtility.classproperty
+            def cluster_id(cls) -> int:
+                """Return cluster id."""
+                return 0x130AFC01
+
+            @ChipUtility.classproperty
+            def attribute_id(cls) -> int:
+                """Return attribute id."""
+                return 0x130A0013
+
+            @ChipUtility.classproperty
+            def attribute_type(cls) -> ClusterObjectFieldDescriptor:
+                """Return attribute type."""
+                return ClusterObjectFieldDescriptor(Type=int)
```

Comment:
Actually, it seems that this attribute is a float32 (from my logs):

```
TypeError: Value 0.0 of type <class 'float'> is invalid for altitude, expected value of type <class 'int'>
```

However, I see that the value currently stays at `0`. I don't think it is useful currently, so I think it is better to  just drop it until it actually provides useful information. Sorry for the back and fourth.

Reply:
In fact, it was set to zero because I hadn't configured it in the Eve application in the first place. Since then, the value is different from `0`

Reply:
Uh I see, this is more a configuration value :thinking: Yeah that make sense, it is probably used to calibrate/improve read out of the pressure sensor.

Ideally we would add it as a configuration then to HA.

Reply:
![image](https://github.com/user-attachments/assets/9be1b62b-15e6-4223-8710-92984a84cbce)